CodeStarterCamp_Week3/main.swift

```diff
@@ -18,18 +18,23 @@ enum Coffee: String {
 }
 
 class Person {
+    var name: String
     var money: Int
     var job: String?
     var house: String?
     
-    init(money: Int) {
+    init(name: String, money: Int) {
+        self.name = name
         self.money = money
     }
     
-    func buyCoffee(paying: Int) {
-        print("지불할 돈은 \(paying)입니다")
-        self.money -= paying
-        print("남은 돈은 \(self.money)입니다")
+    func buyCoffee(paying: Int) -> Int {
+        if self.money-paying < 0 {
```

Comment:
```suggestion
        if self.money - paying < 0 {
```

연산자의 양 옆은 한 칸씩 띄워주세요 🙂